internal/loader/funcdata_invalid.go

```diff
@@ -1,4 +1,4 @@
-// +build !go1.15 go1.19
+// +build !go1.15 go1.20
```

Comment:
Please update the error messages below accordingly.

Reply:
done